servers/list-api/src/dataService/savedItemsService.ts

```diff
@@ -293,7 +293,7 @@ export class SavedItemDataService {
   ) {
     const input = rows.map((row) =>
       Object.keys(row).reduce((obj, key) => {
-        if (row[key] instanceof Date && isNaN(row[key])) {
+        if (isNaN(row[key]) && row[key] instanceof Date) {
```

Comment:
new version of typescript caught this!

Reply:
maybe it's the diff but what is the change here? the ordering of the statements?